### DISCUSSION TITLE
.nominal.associations using theils_u

### DISCUSSION THREAD

**Discussion by shikhagarwal** [shakedzy/dython#98] [Q&A] [answered]
As per the documentation, `if nom_nom_assoc == 'theil'` in `.nominal.associations`, then the output is U(row | col). I spot-checked for my data and turns out that the heat-map outputs `U(col | row)` and not `U(row | col)`. 
```
if nom_nom_assoc == 'theil':
     ji = theils_u(dataset[columns[i]], dataset[columns[j], nan_strategy=_SKIP) #change to ij
     ij = theils_u(dataset[columns[j]], dataset[columns[i]], nan_strategy=_SKIP) #change to ji
```
Could you please clarify. Thank you.

**Comment by shakedzy** [comment, marked as answer]
You might be correct, there's an error in the documentation. It indeed is `U(col | row)`. It seems like I wrote it correctly everywhere else except than in that formula - including the line just before, where it states that _rows are the provided information_. That's a serious face-palm write there. I'll fix the formula. Thanks!
